plugins/gatsby-source-cd-airtable/src/transformers.ts

```diff
@@ -109,6 +112,34 @@ export const transformTags = (airTableTags: AirTableTag[]): Tag[] => {
     .reduce((tags: Tag[], langTags) => [...tags, ...langTags], [])
 }
 
+export function transformEvent(event: AirtableEvent): Event | null {
+  const f = event.fields
+  return {
+    rowId: event.id,
+    name: f.Name,
+    summary: f.Summary,
+    description: f.Description,
+    competenceMap: parseCompetenceMap(f['Competence Map']),
+    startTime: new Date(f['Start Time']),
+    endTime: new Date(f['End Time']),
+    status: f.Status,
+    owner: f.Owner[0],
+    project: f.Project.length > 0 ? f.Project[0] : undefined,
+    tags: f.Tags,
+  }
+}
+
+export function parseCompetenceMap(src: string[]): Record<string, number> {
```

Comment:
Tohle je nestastna transformace, vytvori to nutnost dotazovat se v GraphQL na vsechny kompetence. To znamena, ze by to bud melo doplnit 0 pro "nepritomne", nebo to transformovat na pole { Key: Value } paru.

Neurazi me ani jedna varianta, ale navrhoval bych napsat to defensivnejsim zpusobem, kdy nam zmena tech kompetenci do budoucna nerozbije zadny kod. To jest navrhuju to transformovat na pole { key: value } paru.

Reply:
Nebo mozna lepe to netransformovat tady vubec a naparsovat to takhle az primo v kodu stranky.

Reply:
Dobře – až se k tomu dostanem, asi vyhodím transformaci a nechám to až do stránky.